gammapy/modeling/models/__init__.py

```diff
@@ -66,7 +66,6 @@
     TemplatePhaseCurveTemporalModel,
     TemporalModel,
 )
-from .utils import read_hermes_cube
```

Comment:
this will prevent to load it as `from gammapy.modeling.models import read_hermes_cube`

I think we should rather remove utils from modeling.rst we don't have it for the other subpacakges

```
.. automodapi:: gammapy.modeling.models.utils
    :no-inheritance-diagram:
    :include-all-objects:

```